GA-worksheet.md

```diff
@@ -75,21 +70,21 @@
 
 6. What does each digit of this Hexidecimal Color Code represent? #E78
 
- E represents ...
+ E represents ... Red   
 
- 7 represents ...
+ 7 represents ... Green
 
- 8 represents ...
+ 8 represents ... Blue
 
-7. What is the Hex Color Code for true purple?  #
+7. What is the Hex Color Code for true purple?  #800080
 
-8. _______________ is the programming language that allows websites to be interactive.
+8. JavaScript is the programming language that allows websites to be interactive.
 
-9. This tag allows us to add JavaScript into any webpage? <          >
+9. This tag allows us to add JavaScript into any webpage? <script>
 
-10. What is an event?
+10. What is an event? A event is something that occurs when the user press something
```

Comment:
An event is basically anything that happens on the web page: click, hover, page load, etc. We used the alert in this tutorial to alert us of a "click" event.